src/helpers/api.js

```diff
@@ -3,6 +3,9 @@ const ATTRIBUTES_ENDPOINT = 'attributes.json'
 let d2
 let api
 
+export const getD2 = () => d2
```

Comment:
This area feels weird to me. With it relying on `initApi` being run, modifying a local var and then reexporting that. What do you think? (The d2 export seems unused as far as I could see btw)

Reply:
I've removed them completely, the onSubmit function of the MetaData export page, which was the only one that was using these, is now async